### PR TITLE
Implemented css for header

### LINKED ISSUE #18: css for header
We need to implement css for header

### PR DESCRIPTION
**Pull request by ya-broccoli**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Good

**Review by ghost — APPROVED** [review]
Done

File: css/reset.css [hunk 1]
```diff
@@ -1,38 +1,45 @@
 * {
```
Comment (Maiia2023) [inline css/reset.css:1]: add *:: before and *:: after
```
*,
*:: before,
*:: after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```
```

File: css/reset.css [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  font-weight: normal;
```
Comment (Maiia2023) [inline css/reset.css:14]: font-weight: 400 would b better

**Review by Maiia2023 — APPROVED** [review]

**Pull request merged**